atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -137,6 +152,8 @@ def parse_image_build_config(self, config_file_name):
         section = 'image-build'
         for option in ('name', 'version', 'arches', 'target', 'install_tree'):
             value = config.get(section, option)
```

Comment:
configparser would throw ```NoOptionError``` if the option is not specified. Either we should catch it or verify with  ```config.has_option```